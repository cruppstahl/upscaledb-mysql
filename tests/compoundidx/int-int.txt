DROP TABLE IF EXISTS test;
CREATE TABLE test (id INT NOT NULL PRIMARY KEY,
    c1 INTEGER NOT NULL,
    c2 INTEGER NOT NULL,
    INDEX name (c1, c2)
) ENGINE=upscaledb;
INSERT INTO test VALUES(1, 1, 10);
INSERT INTO test VALUES(2, 2, 20);
INSERT INTO test VALUES(3, 3, 30);
INSERT INTO test VALUES(4, 4, 40);
INSERT INTO test VALUES(5, 5, 50);
SELECT * FROM test;
SELECT * FROM test WHERE id >= 3;
SELECT * FROM test WHERE c1 = 3;
SELECT * FROM test WHERE c2 = 30;
UPDATE test SET c1 = 1 WHERE id = 5;
SELECT * FROM test;
UPDATE test SET c1 = 55 WHERE id = 5;
SELECT * FROM test;
DELETE FROM test WHERE id = 1;
SELECT * FROM test;
