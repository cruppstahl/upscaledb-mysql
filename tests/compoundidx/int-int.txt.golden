--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (id INT NOT NULL PRIMARY KEY,
    c1 INTEGER NOT NULL,
    c2 INTEGER NOT NULL,
    INDEX name (c1, c2)
) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES(1, 1, 10)
--------------

--------------
INSERT INTO test VALUES(2, 2, 20)
--------------

--------------
INSERT INTO test VALUES(3, 3, 30)
--------------

--------------
INSERT INTO test VALUES(4, 4, 40)
--------------

--------------
INSERT INTO test VALUES(5, 5, 50)
--------------

--------------
SELECT * FROM test
--------------

id	c1	c2
1	1	10
2	2	20
3	3	30
4	4	40
5	5	50
--------------
SELECT * FROM test WHERE id >= 3
--------------

id	c1	c2
3	3	30
4	4	40
5	5	50
--------------
SELECT * FROM test WHERE c1 = 3
--------------

id	c1	c2
3	3	30
--------------
SELECT * FROM test WHERE c2 = 30
--------------

id	c1	c2
3	3	30
--------------
UPDATE test SET c1 = 1 WHERE id = 5
--------------

--------------
SELECT * FROM test
--------------

id	c1	c2
1	1	10
2	2	20
3	3	30
4	4	40
5	1	50
--------------
UPDATE test SET c1 = 55 WHERE id = 5
--------------

--------------
SELECT * FROM test
--------------

id	c1	c2
1	1	10
2	2	20
3	3	30
4	4	40
5	55	50
--------------
DELETE FROM test WHERE id = 1
--------------

--------------
SELECT * FROM test
--------------

id	c1	c2
2	2	20
3	3	30
4	4	40
5	55	50
